build.gradle

```diff
@@ -55,13 +55,13 @@ dependencies {
   // SSH
   compile 'com.hierynomus:sshj:0.17.2'
   runtime 'com.jcraft:jzlib:1.0.7'
-  runtime 'org.bouncycastle:bcprov-jdk15on:1.52'
+  runtime 'org.bouncycastle:bcprov-jdk15on:1.54'
```

Comment:
If we have a compile time dep, the runtime dep is superfluous and can/should be removed.